### ISSUE TITLE
Falsely marked as malware

### ISSUE THREAD

**Issue by MarcGuiselin** [MarcGuiselin/wedge#1]
The installer for wedge is marked as malware by the following vendors:

1. Windows smart defender sometimes marks installer as Trojan:Win32/Wacatac.B!ml and quarantines the installer executable
2. Google prevents downloading the file, marking it as dangerous

**Comment by rcmaehl** [comment]
Have you tried compiling with a different version of RustC?

**Comment by MarcGuiselin** [comment]
> Have you tried compiling with a different version of RustC?

No. But that's some really good advice. I remember having to try several different toolchain versions to get the alpha version of wedge [down to one false positive](https://www.virustotal.com/gui/file/41d1a497ebf6171aefa2f190dd44732a7fa3c148fcd5b9ea48907c63a4f22a18).

Tried a few different toolchains, and so far 1.59.0 gets the least false positives:
- 1.58.0 - [8 false positives](https://www.virustotal.com/gui/file/ca5bbfc813a8a1b4028b588589e9452fc953e647cc18a320794801446c73e78f)
- 1.58.1 - [7 false positives](https://www.virustotal.com/gui/file/4c4074038f49de86245de4c9b2adf39d9dbe7c3a8790874ebc7986730c3dfa2f)
- 1.59.0 - [3 false positives](https://www.virustotal.com/gui/file/22729490cc2c72e9d9eb114711f519b7bee16460ab62172574497a09b52a1685) No false positives google and ms!
- 1.60.0 - [7 false positives](https://www.virustotal.com/gui/file/870dfd538f1ca73a49e916e0dfb274d780c4443a9edfd55045e454d08142c512)
- 1.61.0 - [7 false positives](https://www.virustotal.com/gui/file/3d3a4f2ce4cb413c61c17aae9a32baf670da407f9ac9cc267765b6a88fd0f408)
- 1.62.0 - [6 false positives](https://www.virustotal.com/gui/file/ec72e77636e70c41b999e61bd4a325806f2403a292cb7579f569f53e9395d0a7)
- 1.62.1 - [6 false positives](https://www.virustotal.com/gui/file/35c2ed7774e4417a7e1f470e4acaa4118c1a40d26367c5d8226d3ece4ef62723)

Machine-learning based malware detection is a joke.

I've updated the release, and Microsoft defender is not quarantining the file anymore. Unfortunately, google chrome still prevents downloading the file, even though in virustotal google is absolutely fine with it.

**Comment by MarcGuiselin** [comment]
> Tell me about it. I'm honestly going to sell out $400 for an EV Code Signing Certificate come tax season because I'm just plain tired of dealing with them.

You've had more of these kinds of issues than any open source software I've seen, so I don't blame you at all haha. I'd already given up in the past. The odds are definitely stacked up against these kinds of projects, but it's a noble and very necessary fight. Thanks for holding up the torch.

**Comment by MarcGuiselin** [comment]
On my end, I've found that if I roll back far enough I can avoid issues with google safe browsing as well. Using a nearly 4 year old toolchain and very outdated dependencies is far from ideal, but it'll work for the time being.

In the meantime I've reported false positives for the flagged binaries. I've also made several reports to google safe browsing [here](https://safebrowsing.google.com/safebrowsing/report_general/) and [here](https://safebrowsing.google.com/safebrowsing/report_error/) for repository, release pages and download links (objects.githubusercontent.com/...) marked as malware.

**Comment by MatthieuAdler** [comment]
I'm new to all of this, so I apologize if this is not the right place to ask, but how would I determine which path the installer is referring to when it gives the error message "Error on step 4/5 The system cannot find the path specified. (os error 3)"?

**Comment by MarcGuiselin** [comment]
> I'm new to all of this, so I apologize if this is not the right place to ask, but how would I determine which path the installer is referring to when it gives the error message "Error on step 4/5 The system cannot find the path specified. (os error 3)"?

That'd be a new issue, unrelated to this one. I created a new issue thread here: #3
